pkg/ccloudv2/networking.go

```diff
@@ -43,13 +43,13 @@ func (c *Client) CreateNetwork(network networkingv1.NetworkingV1Network) (networ
 	return resp, errors.CatchCCloudV2Error(err, httpResp)
 }
 
-func (c *Client) ListNetworks(environment string) ([]networkingv1.NetworkingV1Network, error) {
+func (c *Client) ListNetworks(environment string, name, cloud, region, cidr, phase, connectionType []string) ([]networkingv1.NetworkingV1Network, error) {
```

Comment:
we can define a struct to make this cleaner

Reply:
Let's keep this as-is actually. It's better to have consistent formatting across our SDK wrapper functions. If you wanted to create a new type in the API spec though, that would be a great way to clean up this SDK!